.gitignore

```diff
@@ -12,5 +13,9 @@ coverage-final.lcov
 npm-debug.log
 yarn.lock
 /_apidoc
-deploy_key
 package-lock.json
+/.grunt/
+/.sync
+deploy_key
+deploy_key.pub
+.DS_Store
```

Comment:
Traditionally we don't add this (`.DS_store`) to a repos `.gitignore` but instead manage these on a machine level (by creating or adding to `git config --global core.excludesfile`)